src/templates/index.hbs

```diff
@@ -1,37 +1,97 @@
-<a href="" onClick="event.preventDefault();bookmarklet.iFrame().init()" class="buttonBookmark">Button Here</a>
+<html>
+<head>
+    <link rel="stylesheet" href="https://cdn.knightlab.com/libs/purpleline/latest/css/purpleline.min.css" type="text/css" />
+    <script type="text/javascript" src="https://cdn.knightlab.com/libs/purpleline/latest/js/purpleline.min.js" ></script>
+       <link rel="icon" type="image/png" href="//cdn.knightlab.com/libs/purpleline/latest/img/favicon.ico">
+</head>
```

Comment:
Can we move the head stuff over to the `default.hbs` file. It just looks a little cleaner that way, also it would be allow us to use purple line in other parts of pullquote without duplicating the script and link tags.